CMakeLists.txt

```diff
@@ -10,11 +12,41 @@ if(${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME})
     set_property(GLOBAL PROPERTY PREDEFINED_TARGETS_FOLDER "CMake")
     set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/bin)
     set(LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR}/lib)
+else()
+    # Check required c++ standard of parent project
```

Comment:
I think we should also set the [CXX_STANDARD_REQUIRED](https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD_REQUIRED.html#prop_tgt:CXX_STANDARD_REQUIRED) variable

Reply:
It is set here: https://github.com/simon-berger/cpr/blob/5205468e7f087ef992f4f1ddab99514968b7ac48/CMakeLists.txt#L50
```
set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Reply:
Sorry, I oversaw that one